Classes/Controller/Backend/NewTenantController.php

```diff
@@ -441,7 +442,7 @@ public function indexAction(): void
      *
      * @access public
      *
-     * @return void
+     * @return ResponseInterface
```

Comment:
This is a recursion from #1045 and should be kept unchanged or removed entirely.
```suggestion
     * @return void
```
Furthermore, since the method is empty, it should declare it's return type in it's signature as well.

Reply:
PHPStan will not accept this method declaring void because it conflict with the method of parent class.
So there are 2 posibilities:
1. declare return as void and keep ignore of error message 
2. declare return as ResponseInterface and remove ignore of error message

Which one do you prefer?

Reply:
Then why not simply remove this method? As far as I can see the parent class' implementation isn't abstract, so we could just let it be inherited...